.nengobones.yml

```diff
@@ -21,7 +21,9 @@ setup_py:
     - matplotlib
     - pytest
   docs_req:
-    - nengo_sphinx_theme>=1.0
+    - nbsphinx>=0.6.0
+    - nengo_sphinx_theme>1.2.2
```

Comment:
Why `>` and not `>=`?

Reply:
Because the `one_page` option is introduced in the current dev version of `nengo-sphinx-theme` (`20.4dev0`), but I didn't want to put a development version in the requirements. Putting a non-development version (like `>=20.4`) wouldn't work because `20.4dev0 < 20.4`.